ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json

```diff
@@ -50,6 +50,26 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "respondent1LiPFinancialDetails",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate",
+          "caseworker-civil-solicitor"
```

Comment:
are we sure we want to allow solicitor to update respondent lip financial details if it comes from a lip?

Reply:
Here add 'CITIZEN-DEFENDANT-PROFILE"

Reply:
Added as suggested